### ISSUE TITLE
480p support?

### ISSUE THREAD

**Issue by arti040** [hoglet67/RGBtoHDMI#325]
I did not find the answer anywhere but is 480p output possible on this device? I'd rather like to use it with standard, black 4x3 LCD than 50 inch TV screen. ;-)

**Comment by IanSB** [comment]
@arti040 

Yes, you can output 640x480 but there is less scope for integer scaling with that resolution (aspect ratios may be wrong depending on the source resolution) so you may have to manually select interpolated scaling.
Note you should only use such a resolution if your monitor is actually physically 640x480. 

Always select the native LCD resolution of your monitor if it isn't selected automatically for best results with integer scaling.
